inc/customizer.php

```diff
@@ -96,7 +96,7 @@ function responsive_font_options() {
  *
  * @return string $styles CSS Styles for use in the Customizer.
  */
-function responsive_get_customizer_styles( $use_cache = true ) {
+function responsive_get_customizer_styles( $use_cache = false ) {
```

Comment:
Can you update this to be back to `true` please?

Reply:
Done and done!